frontend/src/metabase/home/containers/ArchiveApp.styled.tsx

```diff
@@ -17,7 +17,7 @@ export const ArchiveBody = styled.div`
 export const ArchiveBarContent = styled.div`
   display: flex;
   align-items: center;
-  padding: 1rem 4rem;
+  padding: 8px 4rem 7px;
```

Comment:
In my comment I meant using height instead of defining paddings because it is more reliable to changes of components that are rendered inside the footer. If you agree, please change that before merging